packages/foreman/rubygem-concurrent-ruby/rubygem-concurrent-ruby.spec

```diff
@@ -63,10 +63,13 @@ cp -a .%{gem_dir}/* \
 
 %files
 %dir %{gem_instdir}
-%doc %{gem_instdir}/LICENSE.txt
+%doc %{gem_instdir}/LICENSE.md
```

Comment:
Could you use `%license` here while you're at it?

Reply:
updated